governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+- Set Spark Protcol WETH market Reserve Factor to 5% as described below.
+- Set Spark Protocol WETH market Variable Rate Slope 1 to 3% as described below.
+- Decrease Harbor Trade Credit Vault (RWA004-A) Debt Ceiling as detailed below.
+- Increase New Silver (RWA002-A) Debt Ceiling and Stability Fee as described below.
+- A MakerDAO Resolution will be approved to initiate a Clydesdale quarterly surplus return as described below.
+- Distribute **2,000,000 DAI** for Launch Project funding as described below.
+- Distribute **134.1 MKR** for Q2 AVC Member Participation Rewards as described below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Dai Savings Rate Increase
+
+In alignment with the recently approved [Enhanced Dai Savings Rate (EDSR)](https://vote.makerdao.com/polling/QmcTRPLx#poll-detail) system, the following changes to the [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) will be made:
+
+- The DSR will be increased by 4.81% from 3.19% to 8%
+
+### Spark D3M Changes
+
+In preparation for the implementation of the Enhanced DSR policy, Phoenix Labs proposes several changes to the Spark Protocol lending market. Please review the discussion [thread](https://forum.makerdao.com/t/phoenix-labs-proposed-changes-for-spark/21422) to help inform your position before voting.
+
+#### Debt Ceiling Increase
+
+As per this [poll](https://vote.makerdao.com/polling/QmR8cYb1#poll-detail) the following changes to the Spark Protocol DAI Direct Deposit Module (D3M), will be made, if this executive proposal passes:
+
+- The DIRECT-SPARK-DAI [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased by 180 million DAI from 20 million DAI to 200 million DAI
+
+#### Spark Proxy Spell
+
+As per this [poll](https://vote.makerdao.com/polling/QmZyFH21#poll-detail) the following changes to the Spark Protocol DAI Direct Deposit Module (D3M) will be made via Spark Protocol proxy spell defined at [0x443f3f4328553f5f85dFc0BA3D59969708201E14](https://etherscan.io/address/0x443f3f4328553f5f85dFc0BA3D59969708201E14#code).
```

Comment:
Ah. No not correct. The changes via the proxy spell are changes to the Spark protocol itself, not to the Spark D3M. 

The Spark D3M is the thing that exists in Maker core, and mints DAI for the spark protocol to use. The Spark protocol has its own set of parameters (its a fork of Aave) which are changed by Maker governance, but that are external to the core Maker protocol